_docs/javascript-api.md

```diff
@@ -911,6 +911,32 @@ Interceptor.attach(Module.findExportByName("libc.so", "read"), {
 
     -   `depth`: call depth of relative to other invocations
 
+    For eaxmple:
+
+{% highlight js %}
+Interceptor.attach(Module.findExportByName(null, 'read'), {
+    onEnter: function (args) {
+        console.log('Context information: ')
+        console.log("Context  : " + JSON.stringify(this.context))
+        console.log("Return   : " + this.returnAddress)
+        console.log("ThreadId : " + this.threadId)
+        console.log("Depth    : " + this.depth)
+        console.log("Errornr  : " + this.err)
+        console.log("Arg 0    : " + args[0])
```

Comment:
Mixture of `"` and `'` for strings. I would suggest using single-quotes consistently as this is the most common in JS code.